### PR TITLE
Next.js Sample

### PR DESCRIPTION
**Pull request by tnorling**
Adds a sample showing how to use msal-react with Next.js

### CODE REVIEW

File: samples/msal-react-samples/nextjs-sample/pages/profile.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+}
+
+const Loading = () => {
+    return <Typography variant="h6">Authentication in progress...</Typography>
+}
+
+export default function Profile() {
+    const authRequest = {
+        ...loginRequest,
+        redirectUri: "http://localhost:3000"
```
Comment (jasonnutter) [inline samples/msal-react-samples/nextjs-sample/pages/profile.js:45]: This seems to be set in a few places, can it be made the default?

Comment (tnorling) [reply samples/msal-react-samples/nextjs-sample/pages/profile.js:45]: Thanks for calling this out! Updated.

File: samples/msal-react-samples/nextjs-sample/pages/profile.js [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+}
+
+export default function Profile() {
+    const authRequest = {
+        ...loginRequest
+    };
+
+    return (
+        <MsalAuthenticationTemplate 
+            interactionType="popup"
```
Comment (jasonnutter) [inline samples/msal-react-samples/nextjs-sample/pages/profile.js:48]:
```suggestion
            interactionType={InteractionType.Popup} 
```

**Review by jasonnutter — APPROVED** [review]

**Review by jmckennon — APPROVED** [review]
LGTM

**Pull request merged**